examples/ingestron/main.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"encoding/json"
+	"os/signal"
+	"syscall"
+	"time"
+        r "gopkg.in/dancannon/gorethink.v2"
+	"github.com/go-steem/rpc"
+	"github.com/go-steem/rpc/transports/websocket"
+)
+
+func main() {
+	if err := run(); err != nil {
+		log.Fatalln("Error:", err)
+	}
+}
+
+// Set the settings for the DB
+func run() (err error) {
+    Rsession, err := r.Connect(r.ConnectOpts{
+    Addresses: []string{"192.168.194.98:28015","192.168.194.91:28015","192.168.194.145:28015","192.168.194.209:28015"},
+    })
+    if err != nil {
+        log.Fatalln(err.Error())
+    }
+
+// Create a table in the DB
+var rethinkdbname string = "steemit"
```

Comment:
@faddat I don't like the variable name. Should be `dbName` or something. Just use camelCase please...